prometheus-xmpp-alerts

```diff
@@ -147,7 +159,10 @@ async def serve_test(request):
             mto=to_jid,
             mbody='Test message',
             mtype='chat')
-    return web.Response(body='Sent message.')
+    try:
+        return web.Response(body='Sent message.')
+    except slixmpp.xmlstream.xmlstream.NotConnectedError:
```

Comment:
When would this raise a NotConnectedError?

Reply:
This was a bad merge.